scripting/get5/recording.sp

```diff
@@ -136,3 +190,30 @@ void SetCurrentMatchRestartDelay(float delay) {
     mp_match_restart_delay.FloatValue = delay;
   }
 }
+
+public int DemoRequestCallback(Handle request, bool failure, bool requestSuccessful,
+                        EHTTPStatusCode statusCode) {
+  if (failure || !requestSuccessful) {
+    LogError("Demo request failed, HTTP status code = %d", statusCode);
+    char response[1024];
+    SteamWorks_GetHTTPResponseBodyData(request, response, sizeof(response));
+    LogError("Response body: %s", response);
+  } else {
+    LogDebug("Demo request succeeded, HTTP status code = %d", statusCode);
+    char response[1024];
+    SteamWorks_GetHTTPResponseBodyData(request, response, sizeof(response));
+    LogDebug(response);
+    if (g_DemoUploadDeleteAfterCvar.BoolValue && !StrEqual(g_DemoFileNameLastUploaded, "")) {
+      LogDebug("g_DemoUploadDeleteAfterCvar set to true, deleteing the file from the game server.");
+
+      if (FileExists(g_DemoFileNameLastUploaded)) {
+        DeleteFile(g_DemoFileNameLastUploaded);
```

Comment:
Maybe do a `LogError` here if the file deletion fails for whatever reason. `DeleteFile` returns false then.